src/scenes/UserGalleryPage/UserGalleryPage.tsx

```diff
@@ -35,6 +39,19 @@ function UserGalleryPage({ queryRef, username }: UserGalleryPageProps) {
     track('Page View: User Gallery', { username });
   }, [username, track]);
 
+  const router = useRouter();
+  const { settings } = router.query;
+
+  const { showModal } = useModalActions();
+
+  useEffect(() => {
+    if (!settings) return;
```

Comment:
we'll need to check `settings === 'true'`, because query params are strings.

if you visit `jakz?settings=false` this will still trigger the modal :)